backends/xsmm/ceed-xsmm.h

```diff
@@ -11,6 +11,13 @@
 #include <ceed.h>
 #include <ceed/backend.h>
 
+#if ((LIBXSMM_VERSION_MAJOR > 1 || \
+      (LIBXSMM_VERSION_MAJOR == 1 && (LIBXSMM_VERSION_MINOR > 17 || (LIBXSMM_VERSION_MINOR == 17 && LIBXSMM_VERSION_PATCH >= 3710)))))
+#define libxsmm_dispatch_gemm(a, b, c) libxsmm_dispatch_gemm(a, b, c)
+#else
+#define libxsmm_dispatch_gemm(a, b, c) libxsmm_dispatch_gemm_v2(a, b, c)
+#endif
+
```

Comment:
Fun fact, the LIBXSMM main goes from patch 3710 to 3708 to 3710 to 3708 to 3715 to 3716 to 3710 to 3711 to 3719 to 3715 to 3718... so we can't really rely upon the patch level to be sequential

Reply:
🙄